docs/_includes/getting-started/accessibility.html

```diff
@@ -4,11 +4,16 @@ <h1 id="accessibility" class="page-header">Accessibility</h1>
 
   <h3>Skip navigation</h3>
   <p>If your navigation contains many links and comes before the main content in the DOM, add a <code>Skip to main content</code> link before the navigation <a href="http://a11yproject.com/posts/skip-nav-links/">(read why)</a>. Using the <code>.sr-only</code> class will visually hide the skip link, and the <code>.sr-only-focusable</code> class will ensure that the link becomes visible once focused (for sighted keyboard users).</p>
+  <div class="bs-callout bs-callout-danger" id="callout-tables-striped-ie8">
+    <p>Due to a long-standing <a href="https://code.google.com/p/chromium/issues/detail?id=262171" title="Chromium bug tracker - Issue 262171: Focus should cycle from named anchor">bug in Chrome</a> and Internet Explorer, you will need to make sure that the target of your skip link is at least programmatically focusable by adding <code>tabindex="-1"</code>.</p>
```

Comment:
No bug report for IE i could find, but linked to an old article which is still relevant. Also, changed from calling it a bug to shortcoming/bug :)